kubernetes/samples/scripts/create-weblogic-domain/domain-home-in-image/create-domain-inputs.yaml

```diff
@@ -22,6 +22,11 @@ adminServerName: admin-server
 # in a Kubernetes cluster.
 domainUID: domain1
```

Comment:
General comment:  I see changes throughout 'create-weblogic-domain/domain-home-in-image'. Does that also mean we need changes in './create-fmw-infrastructure-domain/domain-home-in-image' or should that be tracked by a different/new JIRA? And are there any other image samples that don't rely on WIT?

Reply:
Yes. I will need to make changes to create-fmw-infrastructure-domain sample too. I get this WIP PR out to make sure that this is the correct approach before moving on to work on the fmw sample. I will also need to update the samples doc too. Those changes and changes in this WIP PR will be included in https://github.com/oracle/weblogic-kubernetes-operator/pull/2290